Pacifique78/ADC
# #168383702 Ft sign up api database

#### What does this PR do?
Add and test sign up with database 

#### Description of Task to be completed
During the signup process, there must be a query of adding that user into the database

#### How should this be manually tested?
See if the user is added to the database when the action is allowed and see what will happen when there is any kind of error.

#### What are the relevant pivotal tracker stories?
#168383702

#### Screenshots (if appropriate)
![Screenshot (34)](https://user-images.githubusercontent.com/50268181/64566944-66186100-d357-11e9-80a8-ed9b61412127.png)

![Screenshot (36)](https://user-images.githubusercontent.com/50268181/64567484-a1675f80-d358-11e9-87b9-e5ba90806c7c.png)
![Screenshot (35)](https://user-images.githubusercontent.com/50268181/64567483-a1675f80-d358-11e9-98be-5bdb34339a67.png)
